simple_generator.py

```diff
@@ -0,0 +1,191 @@
+    return {
+        1: 60,
+        2: 62,
+        3: 64,
+        4: 65,
+        5: 67,
+        6: 69,
+        7: 71,
+        8: 'A',
+        9: 'B',
+        10: 'C',
+        11: 'D',
+        12: 'E',
+        13: 'F',
+        14: 'G'
+    }.get(x, 0)
+
+
+# it is happy accord in do-major
+def write_accord(first_step, order):
+    if first_step == 0:
+        return
+
+    track = 0
+    channel = 0
+    time = order
+    duration = 1
+    MyMIDI.addNote(track, channel, first_step, time, duration, 70)
+    MyMIDI.addNote(track, channel, first_step, time+1, duration, 60)
+    MyMIDI.addNote(track, channel, first_step, time+2, duration, 50)
+
+    MyMIDI.addNote(track, channel, first_step+4, time, duration, 70)
+    MyMIDI.addNote(track, channel, first_step+4, time+1, duration, 60)
+    MyMIDI.addNote(track, channel, first_step+4, time+2, duration, 50)
+
+    MyMIDI.addNote(track, channel, first_step+7, time, duration, 70)
+    MyMIDI.addNote(track, channel, first_step+7, time+1, duration, 60)
+    MyMIDI.addNote(track, channel, first_step+7, time+2, duration, 50)
+
+
+# if the value is zero it will be second without volume
+def write_note(value, order, volume):
+        if value == 0:
+            return
+        if isinstance(value, str):
+            write_accord(value, order)
+            return
+        track = 0
+        channel = 0
+        time = order
+        duration = 1
+        MyMIDI.addNote(track, channel, value, time, duration, volume)
+
+
+def play_music(music_file):
+    """
+    stream music with mixer.music module in blocking manner
+    this will stream the sound from disk while playing
+    """
+    clock = pygame.time.Clock()
+    try:
+        pygame.mixer.music.load(music_file)
+        print("Music file %s loaded!" % music_file)
+    except pygame.error:
+        print("File %s not found! (%s)" % (music_file, pygame.get_error()))
+        return
+    pygame.mixer.music.play()
+    while pygame.mixer.music.get_busy():
+        # check if playback has finished
+        clock.tick(30)
+
+
+def ful_music_play(music_file):
+    print("playing music")
+    # pick a midi music file you have ...
+    # (if not in working folder use full path)
+    freq = 10100  # audio CD quality
+    # So, here was 44100 but it gain very strange messages, but music was in better quality
+    # ALSA lib pcm.c:7963:(snd_pcm_recover) under run occurred
+    bitsize = -16  # unsigned 16 bit
+    channels = 2  # 1 is mono, 2 is stereo
+    buffer = 1024  # number of samples
+    pygame.mixer.init(freq, bitsize, channels, buffer)
+    # optional volume 0 to 1.0
+    pygame.mixer.music.set_volume(0.8)
+    try:
+        play_music(music_file)
+    except KeyboardInterrupt:
+        # if user hits Ctrl/C then exit
+        # (works only in console mode)
+        pygame.mixer.music.fadeout(1000)
+        pygame.mixer.music.stop()
+        raise SystemExit
+
+
+if mode == 's' or mode == 'S':
+    write_sequence(make_music_rand())
```

Comment:
I'll write it here. 
This is too tangled. 
We are using `write_sequence` here that seems to be writing something, but we are not stating what and where. 
It seems that `make_music_rand` returns a sequence that we writing here, and it sort of does, but it ALSO calls `write_seq_ac` in a recursive manner that frustrates the reader.
Doing less for more we can try rework `make_music_rand` to return both sequences and call both `write_seq`s here.
But that would not eliminate the issue with single responsibility - single class/method issue

Reply:
fixed)